torch/distributed/rpc/__init__.py

```diff
@@ -104,18 +121,18 @@ def init_rpc(
                 raise TypeError(
                     f"Could not infer backend for options {rpc_backend_options}"
                 )
-            if backend != BackendType.TENSORPIPE:
+            if backend != BackendType.TENSORPIPE:  # type: ignore[attr-defined]
                 logger.warning(
-                    f"RPC was initialized with no explicit backend but with options "
+                    f"RPC was initialized with no explicit backend but with options "  # type: ignore[attr-defined]
```

Comment:
Not sure, what needs to be ignored on this line
```suggestion
                    f"RPC was initialized with no explicit backend but with options "
```

Reply:
On line 128, the log message referred "{BackendType.TENSORTYPE}". However, mypy reports error on line 126 because it is a multi-line statement.

Reply:
does this help? https://github.com/python/mypy/issues/1032

Reply:
> does this help? [python/mypy#1032](https://github.com/python/mypy/issues/1032)
mypy#1032 is fixed by [this PR](https://github.com/python/mypy/pull/6648) which should only work in Python 3.8, my concern is that this feature is not supported on Python < 3.8.